network/src/topology.rs

```diff
@@ -157,6 +170,26 @@ impl KnownNetwork {
                 connections_g.replace(new_connection);
             }
         }
+
+        // Scope the write lock.
+        {
+            let mut nodes_g = self.nodes.write();
```

Comment:
what does `_g` stand for :thinking:?

Reply:
"guard", not sure if this is a bona fide convention.